using-timescaledb/continuous-aggregates.md

```diff
@@ -139,19 +175,98 @@ PostgreSQL. In addition, TimescaleDB continuous aggregates does not
 currently support the `FILTER` clause even though it is possible to
 parallelize but we might add support for this in a future version.
```

Comment:
We've often had people confuse the `FILTER` keyword with a `WHERE` clause, which is often described as a filter predicate.  Way to clarify?